src/components/SpinnerComponent.js

```diff
@@ -15,14 +15,10 @@ export class SpinnerComponent extends React.Component {
     }
 
     this.state = {
-      show: this.props.hasOwnProperty('show') ? this.props.show : false
+      show: this.props.show || false
```

Comment:
Again, not sure why hasOwnProperty was used on props. There's no reason to do it.